telegram/constants.py

```diff
@@ -2221,6 +2221,54 @@ class MessageType(StringEnum):
     """
 
 
+class NanostarAmount(FloatEnum):
+    """This enum contains constants for ``nanostar_amount`` parameter of
+    :class:`telegram.StarAmount`, :class:`telegram.StarTransaction`
+    and :class:`telegram.AffiliateInfo`.
+    The enum members of this enumeration are instances of :class:`float` and can be treated as
+    such.
+
+    .. versionadded:: NEXT.VERSION
+    """
+
+    __slots__ = ()
+
+    NANOSTAR_VALUE = 1 / 1000000000
```

Comment:
```suggestion
    VALUE = 1 / 1000000000
```
this seems a bit superfluous to me :D similar below